app/src/main/java/com/hedvig/app/feature/insurance/ui/CrossSell.kt

```diff
@@ -70,7 +71,11 @@ fun CrossSell(
             ),
             contentDescription = null,
             contentScale = ContentScale.Crop,
-            modifier = Modifier.fillMaxSize(),
+            modifier = Modifier
+                .fillMaxSize()
+                .clickable(
```

Comment:
Does this make the card have a ripple? 🙂

Reply:
There is a different overload of the `Card` composable that does have an `onClick`, but it is currently annotated with `@ExperimentalMaterialApi` (not sure why, maybe they're likely to change its API).
So if this one doesn't behave as expected maybe that one is the solution.

Reply:
It does make a ripple! :)